dialogue-core/src/main/java/com/palantir/dialogue/core/PinUntilErrorNodeSelectionStrategyChannel.java

```diff
@@ -161,6 +162,12 @@ public void onFailure(Throwable throwable) {
         return maybeResponse;
     }
 
+    @Override
+    public Optional<ListenableFuture<Response>> maybeExecuteOnHost(
+            HostId _hostId, Endpoint _endpoint, Request _request) {
+        return Optional.empty();
+    }
```

Comment:
todo?

Reply:
yeah, I need to implement this in all the strategies, but opted for a quicker route of just putting this in the NodeSelectionStrategyChannel